From f400e72dcc629d658a3b5ccb8b8ab892a7821a8e Mon Sep 17 00:00:00 2001
From: Jingdong Lu <jingdong.lu@windriver.com>
Date: Fri, 25 Nov 2011 15:56:59 +0800
Subject: [PATCH 60/60] omap4: pandaboard: fix wlan support

fetch patch from
http://elinux.org/images/8/87/0001-omap4-pandaboard-wlan-fix.patch

CLK32KG is pseudo regulator, on Blaze/panda it supplies GPS/WIFI/BT/FM/GPS
Drivers should handle it. Keeping it always ON as of now.

Integrated-by: Jingdong Lu <jingdong.lu@windriver.com>
---
 arch/arm/mach-omap2/board-omap4panda.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/arch/arm/mach-omap2/board-omap4panda.c b/arch/arm/mach-omap2/board-omap4panda.c
index ca04657..a537bd8 100644
--- a/arch/arm/mach-omap2/board-omap4panda.c
+++ b/arch/arm/mach-omap2/board-omap4panda.c
@@ -377,6 +377,7 @@ static struct regulator_init_data omap4_panda_vusb = {
 static struct regulator_init_data omap4_panda_clk32kg = {
 	.constraints = {
 		.valid_ops_mask		= REGULATOR_CHANGE_STATUS,
+		.always_on      = true,
 	},
 };
 
-- 
1.7.4.1

